scripts/aggregate-themes/theme-aggregator.js

```diff
@@ -64,11 +71,22 @@ class ThemeAggregator {
 
     const {
       theme: defaultThemeToAggregate,
-      scoped: scopedThemesToAggregate,
+      aggregateDefaultThemeAsScopedTheme,
     } = options;
 
+    let scopedThemesToAggregate = options.scoped || [];
```

Comment:
```suggestion
     scoped,
      aggregateDefaultThemeAsScopedTheme,
    } = options;

    let scopedThemesToAggregate = scoped || [];
```

Reply:
Do we need to update this check for length since the fallback is `[]`? https://github.com/cerner/terra-toolkit/pull/390/files#diff-1672fd55492cdce050f5aa314b17029bR96

Reply:
We can remove it since it defaults to an array.